apps/socketoptions.hpp

```diff
@@ -232,7 +232,8 @@ const SocketOption srt_options [] {
     { "kmrefreshrate", 0, SRTO_KMREFRESHRATE, SocketOption::PRE, SocketOption::INT, nullptr },
     { "kmpreannounce", 0, SRTO_KMPREANNOUNCE, SocketOption::PRE, SocketOption::INT, nullptr },
     { "strictenc", 0, SRTO_STRICTENC, SocketOption::PRE, SocketOption::BOOL, nullptr },
-    { "peeridletimeo", 0, SRTO_PEERIDLETIMEO, SocketOption::PRE, SocketOption::INT, nullptr }
+    { "peeridletimeo", 0, SRTO_PEERIDLETIMEO, SocketOption::PRE, SocketOption::INT, nullptr },
+    { "filter", 0, SRTO_FILTER, SocketOption::PRE, SocketOption::STRING, nullptr }
```

Comment:
A non-representational option name: "filter".
Proposals: packetfilter?

Reply:
My last knowledge on it is that it has been agreed upon already.